src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/AuthDetails.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services;
+
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+
+public class AuthDetails {
+    final UserDetails userDetails;
+    final String userToken;
+    final String serviceToken;
```

Comment:
can we remove the service token please, as mentioned in the previous PR..
we should call generate every time as its cached and will prevent the situation where it expires in between calls

Reply:
sure ... we can refactor this in the upcoming cache token card.
At the moment it represents the token and auth information needed to make the Ccd call.
I hesitate to take it out since it seems a logical bundle.

Reply:
this change is required, serviceToken already has a cache implementation, lets not add code that isn't write by design...
New caching needed is for the userToken

Reply:
Ive created a composite AuthDetails with a UserAuthDetails object which satisfies both our our  requirements :D